marketing-guidelines.md

```diff
@@ -0,0 +1,99 @@
+# OpenTelemetry Marketing Guidelines for Contributing Organizations
+
+OpenTelemetry is a collaboration among end-users, adjacent OSS projects, and
```

Comment:
Please add the header as we do in other files: "this document is work in progress" to reflect the fact that it will never be complete.

Reply:
feels redundant, all policy documents are living documents.

Reply:
hm, I remember we had these notes before. Looks like they were removed. Agree than